manifests/jail.pp

```diff
@@ -29,7 +29,15 @@
   file { "custom_filter_${name}":
     ensure  => file,
     path    => "${config_dir_filter_path}/${name}.conf",
-    content => epp('fail2ban/common/custom_filter.conf.epp'),
+    content => epp('fail2ban/common/custom_filter.conf.epp',
+      {
+        findtime                 => $findtime,
```

Comment:
epp with params \o/

Reply:
No promises I'll get around to it today but I'll try.